README.md

```diff
@@ -1,3 +1,77 @@
 # wodin-proxy
 
-Proxy configuration for wodin. I'll try and get this going for easy cases first, then start thinking about ssl.
+Proxy configuration for wodin. This can bring up [nginx](https://nginx.org/) in either http or https mode, pointing at a number of wodin instances. Generally we'll assume that these are running each on a different container, but they could also be running on the same machine but with a series of ports.
+
+You need a directory, say `sites/` containing small snippets of configuration describing each site:
```

Comment:
I'll update the docs to be more clear here, but so I have it:

* the snippets must be named `<name>.conf`; the `<name>` part is not important but it would make sense to use the same name as the wodin root they correspond to (e.g., `msc-idm-2022.conf` for the configuration for `/msc-idm-2022/`)
* the deploy tool will write these out, they're pretty easy and I don't think any actual configuration is required